test/fixer/sqs_client_test.rb

```diff
@@ -0,0 +1,57 @@
+# encoding: utf-8
+
+require 'minitest_helper'
+
+describe Fixer::SqsClient do
+
+  let(:client) { Fixer::SqsClient.new }
+
+  let(:job_params) do
+    {
+      job: {
+        job_type: 'test',
```

Comment:
We can update this repository to use the single-quote preference we have previously discussed, but I think that @farski has flip flopped and wants everyone using double quotes now :/